pkg/filesystem/fs/fs.go

```diff
@@ -336,6 +336,16 @@ func (fs *Filesystem) RemoveCache(blobDigest string) error {
 		return errors.Wrapf(err, "invalid blob digest from label %s", label.CRILayerDigest)
 	}
 	blobID := digest.Hex()
+	if fs.fsDriver == config.FsDriverFscache {
+		c, err := fs.getSharedDaemon().GetClient()
+		if err != nil {
+			log.L.Warnf("GetClient err %s", err)
+		}
+		// delete fscache blob cache file
+		if err := c.UnbindBlob("", blobID); err != nil {
+			log.L.Warnf("delete blob %s err %s", blobID, err)
```

Comment:
Should return error?

Reply:
yes , we better return error , but this will cause snapshotter can not forward compatible nydusd?

Reply:
You are right, perhaps we should let snapshotter nydusd client is aware or nydusd version to workaround